src/main/java/picard/util/ThreadPoolExecutorUtil.java

```diff
@@ -8,9 +8,10 @@
 public class ThreadPoolExecutorUtil {
     private static final Log log = Log.getInstance(ThreadPoolExecutorUtil.class);
 
-    public static void awaitThreadPoolTermination(final String executorName, final ThreadPoolExecutor executorService) {
+    public static void awaitThreadPoolTermination(final String executorName, final ThreadPoolExecutor executorService,
+                                                  final long minutesBetweenChecks) {
```

Comment:
To make it general I'd use `java.time.Duration`

Reply:
:+1: